packages/zoe/test/unitTests/contracts/zcfTesterContract.js

```diff
@@ -0,0 +1,26 @@
+/* global harden */
+// @ts-check
+
+import { assert } from '@agoric/assert';
+
+/**
+ * Tests ZCF
+ *
+ * @typedef {import('../../../src/zoe').ContractFacet} ContractFacet
+ * @typedef {import('@agoric/ERTP').Amount} Amount
+ * @param {ContractFacet} zcf
+ */
+const makeContract = zcf => {
+  const { brandKeywordRecord, issuerKeywordRecord } = zcf.getInstanceRecord();
+  Object.keys(brandKeywordRecord).forEach(keyword => {
+    // TODO: import tap/tape and do t.equals
+    assert.equal(
+      zcf.getIssuerForBrand(brandKeywordRecord[keyword]),
+      issuerKeywordRecord[keyword],
+    );
+  });
+  return zcf.makeInvitation(() => {}, 'test');
+};
+
+harden(makeContract);
+export { makeContract };
```

Comment:
Is this intended to be called? Is it a replacement for current functionality in the tests?